spark/interpreter/src/test/java/org/apache/zeppelin/spark/NewSparkInterpreterTest.java

```diff
@@ -385,5 +396,7 @@ public void onUpdate(int index, InterpreterResultMessageOutput out) {
               }
             })
     );
+    context.setClient(mockRemoteEventClient);
```

Comment:
Is this a good idea to have a similar test for OldSparkInterpreter too?

Reply:
OldSparkInterpreter already has this kind of testing (but doesn't use mock). https://github.com/apache/zeppelin/blob/master/spark/interpreter/src/test/java/org/apache/zeppelin/spark/OldSparkInterpreterTest.java#L340